Kickstarter-iOS/Library/Optimizely+OptimizelyClientType.swift

```diff
@@ -2,7 +2,18 @@ import Foundation
 import Library
 import Optimizely
 
-extension OptimizelyClient: OptimizelyClientType {}
+extension OptimizelyClient: OptimizelyClientType {
+
+  public func track(eventName: String) {
```

Comment:
Can you put this helper `track` function in the `OptimielyClientType.swift` file with the other helper functions?

Reply:
That was my original thought however, I had an issue calling the track function in the protocol definition from this function.

Reply:
Moved to the `OptimizelyClientType`